projects/ui/src/lib/components/po-field/po-lookup/interfaces/po-lookup-literals.interface.ts

```diff
@@ -29,4 +29,19 @@ export interface PoLookupLiterals {
 
   /** Texto exibido no título da modal. */
   modalTitle?: string;
+
+  /** Texto do span do busca avançada. */
```

Comment:
```suggestion
  /** Texto do link de busca avançada. */
```
Na issue pede para adicionar AVISO na doc que se for passado uma literal muito grande pode quebrar o layout.